.github/workflows/ci.yml

```
@@ -90,7 +90,7 @@ jobs:
         uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --release --locked --features=runtime-benchmarks,ismp
+          args: --release --locked --workspace --exclude integration-tests --features=runtime-benchmarks,ismp
```

I'm not sure we should do this change.

I realize all of the 2412 PRs are currently failing. However, IMO, it's a conscious decision to let them fail. Integration tests failing is a temporary thing until the 2412 release is fixed. So, in order to keep it in our mind, I personally prefer to see the integration tests CI failing so that it reminds me to run it manually on my local machine.

By excluding from the CI, it's easier to let things slip through the cracks and we also lose the reminder to fix once 2412 is fixed.

---

I don't get your point, the CI runs the integration tests (currently not all pop runtimes though) so the CI will still show you that the integration tests don't work

---

fair enough. I did misunderstand some there. I do see that the tests are still run, it's just not included in the initial checks. I am good with that then.